src/v2/view-builder/views/DeviceChallengePollView.js

```diff
@@ -68,8 +68,7 @@ const Body = BaseForm.extend(Object.assign(
           className: 'ul-button button button-wide button-primary',
           title: loc('universalLink.button', 'login'),
           click () {
-            const universalLink = deviceChallenge.href;
-            universalLink && Util.redirectWithForm(universalLink);
+            window.location.href = deviceChallenge.href;
```

Comment:
@nikhilvenkatraman-okta @aarongranick-okta I had to make this change after you reviewed. We tested in the real environment with the authenticator installed. Only window location change with the universal link can launch the app, form get with `Util.redirectWithForm` does not.

Reply:
How are you mocking this in tests so that we don't actually change the URL?

Reply:
We can't really. That's why we test it for real with app in iOS.